agent/util-scripts/pbench-tool-meister-start

```diff
@@ -217,28 +323,28 @@ def kill_redis_server(pid_file):
         raw_pid = pid_file.read_text()
     except Exception:
         # No "pid" to kill
-        return 2
+        return ReturnCode.kill_ret_code(ReturnCode.KILL_READEXC, ret_val)
```

Comment:
No logging on these error paths?

Reply:
At least in all the places that I can see, we have already logged an error that required us to kill the redis server.